addons/portal_rating/static/src/scss/portal_rating.scss

```diff
@@ -56,22 +56,11 @@ $o-w-rating-star-color: #FACC2E;
 /* Star Widget */
 .o-mail-Composer-starCard{
     width: max-content;
-    margin-bottom: 5px;
```

Comment:
This padding of `5px` was wrong, but it was used in both `website_sale` and `website_slides`, thus applying the utility class in `composer_patch.xml` only, will not apply it to `website_slides`. 

I'd suggest applying an mb-2 on the `o-mail-Composer-stars` in `portal_tools.xml`
<img width="690" height="336" alt="image" src="https://github.com/user-attachments/assets/4ff4b4ff-acd7-4420-b9df-6f599a1a1376" />